.buildkite/publish-react-native-ios-artifacts.sh

```diff
@@ -1,19 +1,28 @@
 #!/bin/bash -eu
 
-cd ./ios-xcframework
+# Make Homebrew run a bit faster
+export HOMEBREW_NO_AUTO_UPDATE=1
+export HOMEBREW_NO_INSTALL_CLEANUP=1
+
+echo "--- :arrow_down: Download iOS JS bundle"
+buildkite-agent artifact download bundle/ios/App.js .
 
 echo "--- :rubygems: Setting up Gems"
+cd ./ios-xcframework
```

Comment:
I could have left the `cd` at the start of the script and downloaded the artifact in `..`, but it seemed convoluted.